docs/modules/ROOT/pages/operations/browser-sso.adoc

```diff
@@ -27,102 +27,109 @@ It is strongly advised to use PKCE to ensure security.
-If you are unsure, please consult Neo4j Professional Services to avoid misconfiguration.
-+
-[source, parameters]
-----
-
-{
-	// other discovery entries
-	// e.g. "bolt": "bolt://localhost:7687"
-	//
-	"sso_providers": [
-	 {
-			"id": "keycloak-oidc",  // has to be unique in this file!
-			"name": "Keycloak", // displayed in UI
-			"auth_flow": "pkce",
-      "auth_endpoint": "http://localhost:18080/auth/realms/myrealm/protocol/openid-connect/auth",
- 			"token_endpoint": "http://localhost:18080/auth/realms/myrealm/protocol/openid-connect/token",
- 			"well_known_discovery_uri": "http://localhost:18080/auth/realms/myrealm/.well-known/openid-configuration",
-			"params": {  // can be used for both the auth and the token request
-				"client_id": "account",
-				"redirect_uri": "http://<browser-server-host>:<http-port>?idp_id=keycloak-oidc&auth_flow_step=redirect_uri",
-				"response_type": "code",  // depends on the auth_flow
-				"scope": "openid groups"
-			},
-			"auth_params": { // optional
-				"param_p": "<extra parameter used only for the auth request>"
-			},
-			"token_params": { // optional
-				"client_secret": "<secret-here>", // this may be required by some Idp's and depended on the auth flow.
-				"param_p": "<extra parameter used only for the token request>"
-			},
-			"config": { // optional settings, these allow you to overwrite the defaults
-				"implicit_flow_requires_nonce": false, // Default: false; Desc: Specify if the implicit auth flow requries a nonce in the request
-				"principal": "preferred_username",  // Default: email, otherwise sub; Desc: Optional, in which token claim the user's principal is specified
-				"token_type_principal": "access_token" // Default: access_token; Desc: Which token type is decoded to acquire the specified principal
-				"token_type_authentication": "access_token" // Default: access_token; Desc: Which token type is used as password
-				"code_challenge_method": "S256" // Default is "S256" and it's the only supported method at this moment.
-			}
-		}
-	]
-}
-----
-+
 [IMPORTANT]
 ====
-`redirect_uri` MUST match exactly the redirect_uri specified in the IdP.
+Security information should always be exchanged with encrypted transport, and therefore HTTPS should be used.
+Mixed HTTP/HTTPS flows for single sign-on are not supported.
 ====
 
+Once your SSO provider is configured, you need to configure Neo4j to use OpenID Connect.
+This is done by updating the link:https://neo4j.com/docs/operations-manual/current/configuration/neo4j-conf/#neo4j-conf[`neo4j.conf`^] file according to the instructions in link:https://neo4j.com/docs/operations-manual/current/authentication-authorization/sso-integration/#auth-sso-configure-sso[Operations Manual -> Configure Neo4j to use OpenId Connect^].
+
+Make sure to avoid duplicate entries in the `neo4j.conf` file.
+
+// Optionally, you may set extra logging for the OAuth2 plugin with these settings in the `neo4j.conf` file.
```

Comment:
Maybe check my comment for this part here on the PR in the neo4j-insight repo: https://github.com/neo-technology/neo4j-insight/pull/1896#discussion_r747585254